templates/plugin-bootstrap.mustache

```diff
@@ -12,7 +12,7 @@ if ( ! $_tests_dir ) {
 }
 
 if ( ! file_exists( $_tests_dir . '/includes/functions.php' ) ) {
-	echo "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL;
+	esc_html( "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" ) . PHP_EOL;
```

Comment:
I think to get around the incorrect phpcs `echo` warning probably best to do
```
fwrite( STDERR, "Could not find '$_tests_dir/includes/functions.php', have you run bin/install-wp-tests.sh ?" . PHP_EOL );
```
here, as `esc_html()` not appropriate for shell output.

Reply:
Hi, @gitlost 
As I recall I did not make this change 🙁.
Perhaps it was added when I messed up the commit.
Do you think we can just revert it back to `echo`?

Reply:
Ah okay, it somehow got added in the melee here https://github.com/wp-cli/scaffold-command/pull/118#commits-pushed-9230cb3.

> Do you think we can revert it back to echo?

Yes, that's the simplest thing to do, as contrary to what I said above phpcs doesn't seem to object to it anyway, so please just do that, thanks!